docs/source/howto/data.rst

```diff
@@ -511,9 +511,276 @@ To learn about more advanced queries, please see :ref:`the corresponding topics
 
 .. _how-to:data:organize:
 
-Organizing data
-===============
```

Comment:
@yakutovicha I think you accidentally deleted the section header here. Can you restore that?

Reply:
yes, sorry. Put it back.